app/Core/Database/database.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Application\Core\Database\Database;
+
+// Loading the config
+require_once(ROOT . '/app/config/config.php');
+
+class DatabaseConnection
```

Comment:
nom du fichier DatabaseConnection

Reply:
to change the file's name to DatabaseConnection making the application no more working... I keep it on mind... i'll return on it when i'll have a clearest brain. worked on my actuel branch Making-adminAdmin and i do a commit for all the others camelCase issues...
thanks a lot for all this work of re reading's code

Reply:
Ok it's done : 

<img width="385" alt="Capture d’écran 2022-08-01 à 11 57 53" src="https://user-images.githubusercontent.com/18627668/182123840-37058956-575e-43c5-ad48-4077f3f2b7cd.png">

I named the file DatabaseConnexion to stay in the Englishspoken logic...